xbmc.py

```diff
@@ -690,26 +690,25 @@ def executeJSONRPC(jsonrpccommand):  # NOSONAR
         pass
 
     json_responses = os.environ.get("KODI_STUB_RPC_RESPONSES")
-    if not json_responses:
-        raise ValueError(
-            "Could not find JSON Response folder. Use the environment variable KODI_STUB_RPC_RESPONSES to set one")
-
-    path = "{}.json".format(os.path.join(json_responses, json_data["method"].lower()))
-    if os.path.isfile(path):
-        with io.open(path, mode='r', encoding='utf-8') as fd:
-            stub_content = json.loads(fd.read())
-            if isinstance(stub_content, dict):
-                return json.dumps(stub_content)
-            try:
-                return json.dumps(next(stub.get('response')
-                                       for stub in stub_content
-                                       if json_data.get('params')
-                                       and stub.get('request', {}).get('method') == json_data.get('method')
-                                       and stub.get('request', {}).get('params') == json_data.get('params')))
-            except StopIteration:
-                pass
-
-    return '{"id":1,"jsonrpc":"2.0","result":"OK"}'
+    if json_responses:
```

Comment:
Why did you change this part? If the call to the API is not handled by `JsonRpcApi().handle(json_data)` and no predefined `KODI_STUB_RPC_RESPONSES` path is given then it should just fail right? Or is there a scenario that would make the old behaviour not work?

Reply:
The current behaviour is also to just give OK back if there is no file in the rpc_responses path. I'm okay to always error, but this isn't the current behaviour.m and will probably break a lot of current tests since you would need to implement every JSON RPC value.

Reply:
True